CollapseLauncher/XAMLs/MainApp/Pages/GameSettingsPage.xaml.cs

```diff
@@ -154,12 +153,13 @@ private void InitializeSettings(object sender, RoutedEventArgs e)
                     PageContent.Visibility = Visibility.Collapsed;
                     OverlayTitle.Text = Lang._GameSettingsPage.OverlayNotInstalledTitle;
                     OverlaySubtitle.Text = Lang._GameSettingsPage.OverlayNotInstalledSubtitle;
-
-                    return;
                 }
+                else
```

Comment:
I dont think its necessary to make a new else block just for this and we can just use the else if block above it

Reply:
Set the presence only if the game is not running and not broken.